tests/unit/BitrateTest.ts

```diff
@@ -50,14 +51,22 @@ describe('BitrateTest', () => {
   };
 
   beforeEach(() => {
+    options = {
+      getRTCIceCandidateStatsReport: sinon.stub()
+        .returns(Promise.resolve({
```

Comment:
Should we use `.resolves` instead of `.returns` here? I had issues with `.returns(Promise.reject(...))` and had to use `.rejects` instead, so there may be a difference.

Reply:
Not sure if there's a difference, but I like your suggestion.

Reply:
I changed it to `.resolves`